tabs-and-subtabs/src/components/Header/Header.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import Tab from './../Tab/Tab'
```

Comment:
1. Try using absolute paths instead of relative ones.
    Helps in easier migration in cases of restructuring of files.
    Ref: https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d
2. Keep node module imports separated by a line break from local imports.